nilearn/glm/thresholding.py

```diff
@@ -119,10 +119,10 @@ def cluster_level_inference(
 
     Parameters
     ----------
-    stat_img : Niimg-like object or None, optional
+    stat_img : Niimg-like object or None, default=None
```

Comment:
```suggestion
    stat_img : Niimg-like object
```

If I am reading the function definition correctly this is not optional.
The original `or None` was a mistake.